### PR TITLE
Linux support webdriver factory

### LINKED ISSUE #309: Update WebDriverFactory to support Linux
Need the ability to run Browser tests on a Linux machine locally.  

Need to update the WebDriverFactory to support different operating systems and locations of drivers.

Only provide driver location for windows systems.

### PR DESCRIPTION
**Pull request by FermJacob**
Closes #309 

Not requiring chromedriver.exe path for Linux support
Removing FakerData documentation
Added HowTo on adding chromedriver path

### CODE REVIEW

File: docs/MAQS_5/Utilities/FakerData.md [hunk 1]
```diff
@@ -1,84 +0,0 @@
-# <img src="resources/maqslogo.ico" height="32" width="32"> Faker Data
```
Comment (TroyWalshProf) [inline docs/MAQS_5/Utilities/FakerData.md:1]: Why is this removed - Isn't faker data still in 5?

File: docs/MAQS_5/Selenium/SeleniumFAQ.md [hunk 2]
```diff
@@ -70,4 +70,28 @@ Find the configuration of browsers within the app.config file and define one and
...
+    protected override IWebDriver GetBrowser()
+    {
+        var chromeOptions = new ChromeOptions();
+        chromeOptions.AddArgument("test-type");
+        chromeOptions.AddArguments("--disable-web-security");
+        chromeOptions.AddArguments("--allow-running-insecure-content");
+        chromeOptions.AddArguments("--disable-extensions");
+        chromeOptions.AddArgument("--no-sandbox");
+        chromeOptions.AddArguments("--headless");
+        IWebDriver driver = new ChromeDriver(GetDriverLocation("chromedriver.exe"), chromeOptions);
```
Comment (TroyWalshProf) [inline docs/MAQS_5/Selenium/SeleniumFAQ.md:94]: No command timeout?
SeleniumConfig.GetCommandTimeout();

File: docs/MAQS_5/Selenium/SeleniumFAQ.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ Find the configuration of browsers within the app.config file and define one and
 ## Why doesn't PhantomJS work
```
Comment (TroyWalshProf) [inline docs/MAQS_5/Selenium/SeleniumFAQ.md:14]: Why are you changing V5 docs?

File: docs/MAQS_5/Utilities/FakerData.md (hunk 1 above)
Comment (FermJacob) [reply docs/MAQS_5/Utilities/FakerData.md]: Fixed this issue

File: docs/MAQS_5/Selenium/SeleniumFAQ.md (hunk 2 above)
Comment (FermJacob) [reply docs/MAQS_5/Selenium/SeleniumFAQ.md]: Added in the MAQS 6 documentation

File: docs/MAQS_5/Selenium/SeleniumFAQ.md (hunk 3 above)
Comment (FermJacob) [reply docs/MAQS_5/Selenium/SeleniumFAQ.md]: Fixed

File: docs/MAQS_6/Selenium/SeleniumFAQ.md [hunk 4]
```diff
@@ -70,4 +70,29 @@ Find the configuration of browsers within the app.config file and define one and
...
+    protected override IWebDriver GetBrowser()
+    {
+        var chromeOptions = new ChromeOptions();
+        chromeOptions.AddArgument("test-type");
+        chromeOptions.AddArguments("--disable-web-security");
+        chromeOptions.AddArguments("--allow-running-insecure-content");
+        chromeOptions.AddArguments("--disable-extensions");
+        chromeOptions.AddArgument("--no-sandbox");
+        chromeOptions.AddArguments("--headless");
+        IWebDriver driver = new ChromeDriver(GetDriverLocation("chromedriver.exe"), chromeOptions, SeleniumConfig.GetCommandTimeout());
```
Comment (TroyWalshProf) [inline docs/MAQS_6/Selenium/SeleniumFAQ.md:95]: Why not use 
WebDriverFactory.GetDefaultChromeOptions()
to get the options?
